pkg/policy/existing.go

```diff
@@ -34,8 +34,13 @@ func (pc *PolicyController) processExistingResources(policy kyverno.ClusterPolic
 		}
 
 		// skip reporting violation on pod which has annotation pod-policies.kyverno.io/autogen-applied
-		if skipPodApplication(resource, logger) {
-			continue
+		ann := policy.GetAnnotations()
+		if _, ok := ann[engine.PodTemplateAnnotation]; ok {
+			if ann[engine.PodTemplateAnnotation] == "none" {
+				if skipPodApplication(resource, logger) {
+					continue
+				}
```

Comment:
If `engine.PodTemplateAnnotation == none`, we should not skip policy application.